Display.cs

```diff
@@ -43,7 +43,7 @@ public static class Display {
         public static Image CreateImage(double width, double height, string data, double hScale, double vScale, double transform) => Controller.CreateImage((int)width, (int)height, data, (int)hScale, (int)vScale, (Image.Transform)transform);
         public static void Image(object img, double x, double y) => Controller.DrawImage((Image)img, (int)x, (int)y);
         public static void Show() => Controller.Show();
-        public static void Color(uint c) => color = c;
+        public static void Color(double c) => color = (uint)c;
```

Comment:
What is the benefit of changing this from uint to double?

Reply:
I tested quickly and the compiler is doing the same as the original interpreter, if you pass a double as an argument to something expecting an int it will be truncated.

Reply:
I see, the issue is on the C# side. I was looking at the DUE interface which handles this.

Reply:
Yeb, problem on C# side, not DUE.